src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -205,17 +289,48 @@ function PolicyAccountingPage({policy, connectionSyncProgress}: PolicyAccounting
                                     childrenStyles={styles.pt5}
                                 >
                                     <MenuItemList
-                                        menuItems={menuItems}
+                                        menuItems={connectionsMenuItems}
                                         shouldUseSingleExecution
                                     />
+                                    {otherIntegrationsItems && otherIntegrationsItems?.length > 0 && (
+                                        <CollapsibleSection
+                                            title={translate('workspace.accounting.other')}
+                                            wrapperStyle={styles.pr3}
+                                            titleStyle={[styles.textNormal, styles.colorMuted]}
+                                        >
+                                            {otherIntegrationsItems.map((integration) =>
+                                                integration?.icon ? (
```

Comment:
If we don't plan to use MenuItemList, do we really need an if else based on `integration?.icon` ?

```
<MenuItem
    icon={integration?.icon}
    iconType={integration?.icon ? CONST.ICON_TYPE_AVATAR : undefined}
    ...
```
   

```